src/Input/Input.js

```diff
@@ -26,7 +26,7 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
       '&:after': {
         backgroundColor: palette.primary.A200,
         left: 0,
-        bottom: -1,
+        bottom: 0,
```

Comment:
Yea this was a good point. So I looked into this and because I changed the position of the underline div I had to set it to `-2` to get the regression tests passing.